2-mandatory/3-financial-times.js

```diff
@@ -15,27 +25,68 @@ function potentialHeadlines(allArticleTitles) {
 */
 function titleWithFewestWords(allArticleTitles) {
     // TODO
+
+    let ascesndingTitles = allArticleTitles.sort((a, b) => a.length - b.length);
+
+
+    return ascesndingTitles[0];
 }
 
 /*
     The editor of the FT has realised that headlines which have numbers in them get more clicks!
     Implement the function below to return a new array containing all the headlines which contain a number.
     (Hint: remember that you can also loop through the characters of a string if you need to)
 */
+// const ARTICLE_TITLES = [
+//     "Streaming wars drive media groups to spend more than $100bn on new content",
+//     "Amazon Prime Video India country head: streaming is driving a TV revolution",
+//     "Aerospace chiefs prepare for bumpy ride in recovery of long-haul flights",
+//     "British companies look to muscle in on US retail investing boom",
+//     "Libor to take firm step towards oblivion on New Year's Day",
+//     "Audit profession unattractive to new recruits, says PwC boss",
+//     "Chinese social media users blast Elon Musk over near miss in space",
+//     "Companies raise over $12tn in 'blockbuster' year for global capital markets",
+//     "The three questions that dominate investment",
+//     "Brussels urges Chile's incoming president to endorse EU trade deal",
+// ];
+
 function headlinesWithNumbers(allArticleTitles) {
     // TODO
+    let result = [];
+
+    for (let title of allArticleTitles) {
+        for (let characters of title) {
```

Comment:
Here I agree with Monika, it works and is great but if you are looking to improve, using character single helps tell the story of what the code is doing a little better :)